docs/plot_result.rst

```diff
@@ -0,0 +1,34 @@
+####################
+Plotting fit results
+####################
+
+Plotting
+========
+
+The saved results from a PAHFIT run can be plotted using the ``plot_pahfit``
+script.  By default, the plot is displayed.  To save the plot instead,
+use the "--savefig" command line option.
```

Comment:
The quotes in --savefig, output some weird characters in the http://www.stsci.edu/~kgordon/pahfit/plot_result.html page.

Reply:
Can you be more specific?  That page looks fine to me.  Maybe a screenshot to show me how it looks to you?

Reply:
![sc1](https://user-images.githubusercontent.com/33009979/53662542-1f6fde80-3c31-11e9-9c87-5bd52f9d52e7.png)

Reply:
Hopefully fixed in the next commit.